src/main.rs

```diff
@@ -49,6 +49,22 @@ fn main() -> Result<(), String> {
         }
     }
 
+    if let Swww::Init { .. } | Swww::Img(_) | Swww::Restore(_) | Swww::Clear(_) = &swww {
```

Comment:
I don't think we need this if statement. We should **always** make sure the daemon is running before sending in any requests to it.

Reply:
That sounds like a good idea!